js/activities.js

```diff
@@ -17,24 +19,24 @@ document.addEventListener("DOMContentLoaded", async () => {
       cCard += clickGen(activity, colorForTitles[i], colorForButtons[i]);
       i++;
     });
-  } catch {
-    console.error();
+  } catch(error) {
+    console.error(error);
   }
   document.getElementById("unclickables").innerHTML = ucCard;
   document.getElementById("clickables").innerHTML = cCard;
 });
 
 const unclickGen = (activity) => {
   return `<div class='bg-white rounded-2xl w-full p-5 pt-8 border-[0.2px] border-[#787676] transition-all drop-shadow-lg hover:drop-shadow-2xl cursor-default flex flex-col gap-4  items-center h-full ${activity.title == "Cake Cutting" ? " sm:hidden lg:flex " : ""}'>
-        <img src="${activity.image}" alt="${activity.title}" class="h-20 w-20">
+        <img src="${baseActivitiesURL+activity.image}" alt="${activity.title}" class="h-20 w-20">
```

Comment:
Its well. Would recommend you to reformat code before sending PRs with IDE or any other formatter plugins.

It will add readability and cleanliness in the code files.